src/Build/Evaluation/ItemSpec.cs

```diff
@@ -300,22 +297,22 @@ public virtual IMSBuildGlob ToMSBuildGlob()
 
         protected virtual IMSBuildGlob CreateMsBuildGlob()
         {
-            return Globbing.MSBuildGlob.Parse(ProjectPath, ItemSpecFragment.Unescape());
+            return Globbing.MSBuildGlob.Parse(ProjectDirectory, ItemSpecFragment.Unescape());
```

Comment:
@cdmihai Can you help me understand this? This was passing the project file name - but it appears that MSBuildGlob expects a "root" path. How is this supposed to work?

Reply:
We talked offline, it works accidentally.